posts/2022-08-04-beta-release-post.adoc

```diff
@@ -0,0 +1,267 @@
+---
+layout: post
+title: "Updates to MicroProfile OpenAPI, Jakarta JSON Binding and Processing and more exciting new features in Open Liberty 22.0.0.9-beta"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/reece-nana
+author_github: https://github.com/reece-nana
+seo-title: Updates to MicroProfile OpenAPI, Jakarta JSON Binding and Processing and more exciting new features in Open Liberty 22.0.0.9-beta - OpenLiberty.io
+seo-description: DESCRIPTION
+blog_description: DESCRIPTION
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= TITLE
+Reece Nana <https://github.com/reece-nana>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+Open Liberty 22.0.0.9-beta has multiple new features such as passwordUtilities-1.1 which no longer forcefully federtaes stand-alone user registries. RESTful Web Services 3.1 is now available in Open Liberty Betas! The MicroProfile OpenAPI will be useful to test out the API during development or for using the API in production. Furthermore, Jakarta's JSON Processing specification has been updated with enhancements for handling duplicate keys when parsing JSON. Jakarta EE 10 Expression Language 5.0 includes a number of new features and specification clarifications and support has been added for Build Compatibile Extensions in CDI 4.0!
```

Comment:
```suggestion
Open Liberty 22.0.0.9-beta brings you the hot-press Jakarta EE 10 Core Profile. Jakarta EE 10 is the first release that provides some updates to the Jakarta EE specifications since Java EE 8 in 2017. While Jakarta EE 10 release is around the corner, Open Liberty has consumed many of the Jakarta EE 10 specifications and put them in the 22.0.0.9-beta.  In this particular beta the features that make up the new Jakarta EE 10 Core Profile are made available: 

- Jakarta RESTful Web Services 3.1: support multipart/form-data.
- Jakarta JSON-B 3.0: deserialization of null to JsonValue.NULL, default PropertyVisibilityStrategy Implemenation etc.
- Jakarta JSON-P 2.1: handling duplicated keys; defined more interfaces for primitives, lists and objects 
- Jakarta Annotation 2.1: allowing `@Priority` to be used on any program elements; adding `@Nullable` and `@NotNull`
- Jakarta Interceptors 2.1: picking up the changes from Annotation 2.1
- Jakarta CDI 4.0: defining new extension APIs; removing reflections; adding new reflection-free language model; empty beans.xml means that only annotated classes are beans; removal of deprecated APIs

In addition to the above feature, it has multiple new features such as passwordUtilities-1.1 which no longer forcefully federtaes stand-alone user registries. The MicroProfile OpenAPI will be useful to test out the API during development or for using the API in production. Jakarta EE 10 Expression Language 5.0 includes a number of new features and specification clarifications!
```